samples/DataSourceWithProcessingAndExtensions/Sample/Helpers/Jsonbinary.cs

```diff
@@ -0,0 +1,41 @@
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace DataExtensionsSample.Helpers
+{
+    public static class JsonBinary
+    {
+        public static byte[] ToBinary(object value)
+        {
+            byte[] result = null;
+
+            using (var stream = new MemoryStream())
+            {
+                using (var writer = new BinaryWriter(stream, Encoding.Default, true))
+                {
+                    writer.Write(JsonConvert.SerializeObject(value));
+                    result = stream.ToArray();
+                }
+            }
+
+            return result;
+        }
+
+        public static T ToObject<T>(byte[] value)
+        {
+            string jsonValue = string.Empty;
+
+            using (var stream = new MemoryStream(value))
+            {
+                using (var reader = new BinaryReader(stream, Encoding.Default, true))
+                {
+                    jsonValue = reader.ReadString();
+                }
+            }
+
+            return JsonConvert.DeserializeObject<T>(jsonValue);
+        }
+    }
+
+}
```

Comment:
This file is the same as the other Jsonbinary.cs. Why can't we just add a using statement and use that one? (Or the other way around).

Either way, if we're keeping this file, can we capitalize the "B" in the filename?